doc/source/whatsnew/v0.16.1.txt

```diff
@@ -18,6 +18,7 @@ Enhancements
 ~~~~~~~~~~~~
 
 - Added ``StringMethods.capitalize()`` and ``swapcase`` which behave as the same as standard ``str`` (:issue:`9766`)
+- Added ``StringMethods`` (.str accessor) to ``Index`` (:issue:`9068`)
```

Comment:
It would be nice to add an example or two here, both to show off the new feature and to illustrate the behavior with boolean output.